runelite-client/src/main/java/net/runelite/client/game/ItemManager.java

```diff
@@ -478,12 +480,13 @@ public AsyncBufferedImage getImage(int itemId, int quantity, boolean stackable)
 	 * @param itemId item id
 	 * @param itemQuantity item quantity
 	 * @param outlineColor outline color
+	 * @param thickness thickness of outline
 	 * @return image
 	 */
-	private BufferedImage loadItemOutline(final int itemId, final int itemQuantity, final Color outlineColor)
+	private BufferedImage loadItemOutline(final int itemId, final int itemQuantity, final Color outlineColor, final int thickness)
```

Comment:
Maybe instead of adding a breaking change to this API, add an overload with the default being 1 thickness.